stacks-signer/src/runloop.rs

```diff
@@ -189,10 +136,9 @@ impl RunLoop {
             warn!("No registered signers found for reward cycle {reward_cycle}.");
             return Ok(None);
         }
-        Ok(Some(Self::parse_nakamoto_signer_entries(
-            &signers,
-            self.config.network.is_mainnet(),
-        )))
+        let entries =
+            ParsedSignerEntries::parse(self.config.network.is_mainnet(), &signers).unwrap();
```

Comment:
Just a nit: seems a bit odd to call something "ParsedSignerEntries::parse" feel like it should be "SignerEntries::parse" or "ParsedSignerEntries::new" or "ParsedSignerEntries::from_raw_signer_entries" but feel free to ignore.

Reply:
Renamed to SignerEntries in [c50a930](https://github.com/stacks-network/stacks-core/pull/4481/commits/c50a93088d7c0261f1dbe31ab24b95028a038447)